loguru.hpp

```diff
@@ -1281,6 +1281,19 @@ This will define all the Loguru functions so that the linker may find them.
 
 namespace loguru
 {
+#ifdef LOGURU_WITH_FILEABS
+	struct FileAbs
+	{
+		pthread_mutex_t lock = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
All logging is already protected by the global `s_mutex`, so this lock is redundant afaict.